benchmarks/conftest.py

```diff
@@ -1,2 +1,7 @@
+import sys
+
+sys.path.append("../test")
```

Comment:
I added this to let me import `get_available_devices` from `_utils_internal`.

We could alternatively accept a small amount of duplication and just copy / paste it?

Reply:
We can copy yes, no worry
I'd rather have that than messing with relative imports (fbcode will complain)